.github/workflows/ci.yml

```diff
@@ -22,7 +22,7 @@ jobs:
                   RESULT=0
                   npm test || RESULT=$?
                   if [ $RESULT -ne 0 ]; then
-                    cat hardhat.log
+                    echo "Tests failed with exit code $RESULT"
```

Comment:
Probably still makes sense to show logs here, would be useful for debugging (e.g. both new message and then cat hardhat.log should be fine)